xds/src/main/java/io/grpc/xds/internal/rbac/engine/AuthorizationDecision.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2020 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.xds.internal.rbac.engine;
+
+import com.google.common.collect.ImmutableList;
+import java.lang.StringBuilder;
+import java.util.List;
+
+/** 
+ * The AuthorizationDecision class holds authorization decision 
+ * returned by CEL Evaluation Engine. 
+ */
+public class AuthorizationDecision {
+  /** The Decision enum represents the possible decisions outputted by CEL Evaluation Engine.*/
+  public enum Decision {
```

Comment:
Will be good avoid repetition of `Decision` in `AuthorizationDecision.Decision` ? Just a suggestion.

Reply:
The enum `Decision` is only declared in class `AuthorizationDecision`, but it's used in `AuthorizationEngine` when generating an `AuthorizationDecision` object.
Which repetition did you mean exactly?

Reply:
I mean the word `Decision` is repeated in fully qualified `AuthorizationDecision.Decision` . It is not a big deal but it will be nice to avoid such things for better readability and intuitiveness. So the `Decision` enum can be called `Outcome` or `Output` or just `Value`?

Reply:
Yeah good point. I think maybe we can call it `Output`?

Reply:
Done :)